src/FsUnit.NUnit/CustomConstraints.fs

```diff
@@ -17,6 +17,4 @@ module CustomConstraints =
                 let result = Common.isOfCase this.Expected actual
                 ConstraintResult(this, actual, result)
             else
-                let actualType = actual.GetType()
-                do printf $"Got a {actualType.Name}"
```

Comment:
Should not it be inside the error message?

Reply:
Gotcha.